deployment/helm/balloons/values.yaml

```diff
@@ -54,15 +56,16 @@ config:
 plugin-test:
   enableAPIs: false
 
-hostPort: 8891
-
 resources:
```

Comment:
This is going to break the metric expose. Because, you don't have a default value for `port.host` field. Meaning, if you set `prometheusExport: true`, you don't get any metric because the port is closed.

Reply:
> This is going to break the metric exposure. Because, you don't have a default value for `port.host` field. Meaning, if you set `prometheusExport: true`, you don't get any metric because the port is closed.

Why would it break it ? Metric exposure does not need any port mapping as this PR switches to prometheus autodiscovery by annotation. Then this PR also removes that metrics-dedicated `hostPort` Helm value altogether. It is replaced with an optional `ports` array. That is a generic port mapping setting and defaults to `nil`. In the daemonset template we evaluate `ports` with a `range`, emitting the necessary corresponding `name`, `hostPort` and `containerPort` triplet `port` entry.

So if metric exposure is now broken I don't see how, neither could I reproduce it in my local tests. Can you elaborate more on that ?